### PR TITLE
Tracing: Add ability to write trace to metrics query

### PR DESCRIPTION
**Pull request by connorlindsey**
**What this PR does / why we need it**:
Lets users write the query to be run in the trace to metrics link
image

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by joey-grafana — APPROVED** [review]
LGTM 👍

File: docs/sources/datasources/zipkin.md [hunk 1]
```diff
@@ -40,6 +40,15 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
 
 ![Trace to logs settings](/static/img/docs/explore/trace-to-logs-settings-8-2.png 'Screenshot of the trace to logs settings')
 
+### Trace to metrics
+
+> **Note:** This feature is behind the `traceToMetrics` feature toggle.
+
+This is a configuration for the experimental trace to metrics feature. Select the target Prometheus data source and enter the desired query.
```
Comment (brendamuir) [inline docs/sources/datasources/zipkin.md:47]:
```suggestion
To configure trace to metrics, select the target Prometheus data source and enter the desired query.
```

File: docs/sources/datasources/zipkin.md [hunk 2]
```diff
@@ -40,6 +40,15 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
...
+
+-- **Data source -** Target data source.
+-- **Query -** The query which will run when navigating from a trace to the metrics data source.
```
Comment (brendamuir) [inline docs/sources/datasources/zipkin.md:50]:
```suggestion
-- **Query -** Query that runs when navigating from a trace to the metrics data source.
```

**Review by brendamuir — CHANGES_REQUESTED** [review]

**Comment by connorlindsey** [comment]
@brendamuir Thanks for the suggestions, I've applied them to the 3 sets of docs.

**Review by brendamuir — APPROVED** [review]

**Pull request merged**